src/data/demo-shopping-cart/promocodes.ts

```diff
@@ -0,0 +1,19 @@
+export const promoCodes = [
+    "HOT-COURSE",
+    "NO-PYTHON",
+    "JAVA-FOR-BOOMERS",
+    "15-PERCENT-FOR-CSS",
+    "HelloThere",
+    "5-PERCENT-FOR-UTILS",
+    "10-PERCENT-FOR-REDEEM"
+  ];
+
+  export const promoCodeDiscountMap: Record<string, number> = {
```

Comment:
Коль мы используем тайпскрипт, то надо его использовать :)

```suggestion
  ] as const;
  
 type PromoCodes = typeof promoCodes[number];

  export const promoCodeDiscountMap: Record<PromoCodes, number> = {
```

Reply:
Applied the suggested change, thank you!